src/query/tsdb/remote/codecs.go

```diff
@@ -181,7 +185,7 @@ func encodeFetchOptions(options *storage.FetchOptions) (*rpc.FetchOptions, error
 
 	result.AggregatedOptimized = aggOpt
 	if v := options.RestrictFetchOptions; v != nil {
-		restrict, err := v.Proto()
+		restrict, err := encodeRestrictFetchOptions(v)
```

Comment:
Do we want to remove the "Proto()" method on `RestrictFetchOptions` now that it's no longer being used?